Description: Use stream
--- a/test/trackerstream.js
+++ b/test/trackerstream.js
@@ -1,7 +1,7 @@
 'use strict'
 var test = require('tap').test
 var util = require('util')
-var stream = require('readable-stream')
+var stream = require('stream')
 var TrackerStream = require('../index.js').TrackerStream
 var testEvent = require('./lib/test-event.js')
 
--- a/tracker-stream.js
+++ b/tracker-stream.js
@@ -1,6 +1,6 @@
 'use strict'
 var util = require('util')
-var stream = require('readable-stream')
+var stream = require('stream')
 var delegate = require('delegates')
 var Tracker = require('./tracker.js')
 
